iso/Dockerfile

```diff
@@ -20,6 +20,10 @@ FROM boot2docker/boot2docker:1.12.3
 RUN echo "1.12.3" > $ROOTFS/etc/version
 ENV DOCKER_SHA a3bb88649bcc47775c29dab7d95709f4dda04d0e
 
+# Add the /etc/os-release file
+RUN sed -i '/^b2dVersion/,/^EOOS/d' /make_iso.sh
```

Comment:
@hferentschik  make_iso.sh has code which overwrites the /etc/os-release file and `make_iso.sh` is called end of the Dockerfile.  So if I do not remove the code and add the os-release file , it will be overwritten by the make_iso.sh script.